cluster/gce/gci/mounter/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TAG?=v1
+REGISTRY?=gcr.io/google_containers
+IMAGE=gci-mounter
+
+all: container
+
+container:
+	docker build --pull -t ${REGISTRY}/${IMAGE}:${TAG} .
```

Comment:
If you decide to do this you'd have to change push to be a two step process as well:
```MAKE
	gcloud docker -- push ${REGISTRY}/${IMAGE} # Push image tagged as latest to repository
	gcloud docker -- push ${REGISTRY}/${IMAGE}:${TAG} # Push version tagged image to repository (since this image is already pushed it will simply create or update version tag)
```